CHANGELOG.adoc

```diff
@@ -44,6 +44,8 @@ See `PapayaExtensionRegistry.tsx` for an example of how rendering a project subt
 It is not an API breaking change but a breaking change from the end-user point of view.
 - https://github.com/eclipse-sirius/sirius-web/issues/4500[#4500] [sirius-web] Remove the tight coupling between `SemanticData` and `Project` by removing `SemanticData#project` along with the column `project_id` from the `semantic_data` table
 - https://github.com/eclipse-sirius/sirius-web/issues/4501[#4501] [browser] `IReferenceWidgetRootCandidateSearchProvider` API has been renamed to `IModelBrowserRootCandidateSearchProvider` and moved to the `sirius-components-collaborative-browser` module.
+- https://github.com/eclipse-sirius/sirius-web/issues/4372[#4372] [sirius-web] The identifier used for editingContextId is now semanticDataId instead of the projectId.
```

Comment:
Since this will be a MASSIVE change for all downstream application which may be using `editingContextId` as the `projectId` some additional information will be necessary to explain them how to proceed with this breaking change.